src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -494,23 +508,22 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 await singleton.AcquireAsync(functionCancellationTokenSource.Token);
             }
 
-            // Check to see if there are any filters
-            var jobInstance = invoker.CreateInstance();
+            object jobInstance = instance.JobInstance;
             parameterHelper.ObjectInstance = jobInstance;
             using (jobInstance as IDisposable)
             {
-                List<IFunctionInvocationFilter> filters = GetFilters(instance.FunctionDescriptor, jobInstance);
-
                 // Create a source specifically for timeouts
                 using (CancellationTokenSource timeoutTokenSource = new CancellationTokenSource())
                 {
                     TimeoutAttribute timeoutAttribute = instance.FunctionDescriptor.TimeoutAttribute;
                     bool throwOnTimeout = timeoutAttribute == null ? false : timeoutAttribute.ThrowOnTimeout;
-                    var timer = StartFunctionTimeout(instance, timeoutAttribute, timeoutTokenSource, traceWriter, logger);
+                    var timer = StartFunctionTimeout(instance, timeoutAttribute, timeoutTokenSource, traceWriter, _logger);
```

Comment:
>StartFunctionTimeout [](start = 32, length = 20)

The new EH Filter won't be covered by this timeout. (Although neither is parameter binding  - so maybe this timeout is just too low in the stack).